config/forms/2021_2022.json

```diff
@@ -2009,7 +2207,7 @@
                   "hhmemb": 8
                 }
               ]
-            },
+            },     
```

Comment:
Random extra whitespace? Our `.editorconfig` is set to trim this automatically on save but might need configuring on yours.